packages/fgtclb/academic-projects/Resources/Private/Partials/Project/Items.html

```diff
@@ -5,18 +5,16 @@
 >
     <f:if condition="{projects}">
         <f:then>
-            <div class="project-list">
+            <div class="ace-list">
```

Comment:
Do we need to mention this or take this as breaking, due to having this styled in projects without overriding the partial ?